plugin-localization/README.md

```diff
@@ -20,7 +20,7 @@ repositories {
 
 // In the app build.gradle file
 dependencies {
-    implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-localization:0.6.0'
+    implementation 'mapbox-android-plugin-localization-v7:0.7.0'
```

Comment:
Whoops, I'm going to fix it up in https://github.com/mapbox/mapbox-plugins-android/pull/810